SIT.Manager/Services/ManagerConfigService.cs

```diff
@@ -12,15 +12,20 @@ internal sealed class ManagerConfigService : IManagerConfigService
 {
     private readonly ILogger<ManagerConfigService> _logger;
 
-    private ManagerConfig _config = new();
-    public ManagerConfig Config
-    {
-        get => _config;
-        private set { _config = value; }
-    }
+    public ManagerConfig Config { get; private set; } = OperatingSystem.IsLinux()
+        ? new LinuxConfig()
+        : new ManagerConfig();
```

Comment:
I think I'd like @artehe 's opinion on this. Idk if I groove with doing things this way as it would mean any linux code needing to access linux options will need to cast the manager config to a linux config beforehand

Reply:
I do agree.

Would a far simpler option not be to just create a class called something like LinuxWineConfig and then just add that as a property to the ManagerConfig so that we don't have to handle two different types of manager config?